include/openssl/ocsp.h

```diff
@@ -180,6 +185,30 @@ OPENSSL_EXPORT int OCSP_basic_verify(OCSP_BASICRESP *bs, STACK_OF(X509) *certs,
 OPENSSL_EXPORT OCSP_CERTID *OCSP_cert_to_id(const EVP_MD *dgst, const X509 *subject,
                                             const X509 *issuer);
 
+// OCSP_parse_url parses an OCSP responder URL and returns its component parts.
+// The url argument must be a null-terminated string containing the URL to be
+// parsed. The other arguments are pointers to variables that will be set to the
+// parsed components of the URL:
```

Comment:
This should cal out the pointers point to new memory that has a copy of the value and it's the caller's responsibility to free them.